### PR TITLE
add select carplay mode menu

### PR DESCRIPTION
**Pull request by nnngrach**
https://github.com/osmandapp/OsmAnd-Issues/issues/483

video:
https://www.dropbox.com/s/oxe3l19r714eeoa/Screen%20Recording%202021-06-19%20at%2010.09.21.mov?dl=0

### CODE REVIEW

File: Resources/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -191,6 +191,8 @@
 "favorites_empty" = "You have not saved any favorite points";
 
 "carplay_active_message" = "Map is displayed via CarPlay";
+"carplay_profile" = "CarPlay profile";
+"carplay_profile_descr" = "Profile that will be used after connection to Carplay.";
```
Comment (PaulStets) [inline Resources/en.lproj/Localizable.strings:195]: ...used while connected to CarPlay

File: Sources/CarPlay/OACarPlayMapViewController.mm [hunk 2]
```diff
@@ -13,6 +13,7 @@
 #import "OAMapPanelViewController.h"
 #import "OANativeUtilities.h"
 #import "OAMapViewTrackingUtilities.h"
+#import "OAAppSettings.h"
```
Comment (PaulStets) [inline Sources/CarPlay/OACarPlayMapViewController.mm:16]: Why is it here?

File: Sources/Helpers/OAAppSettings.h [hunk 3]
```diff
@@ -679,6 +679,8 @@ typedef NS_ENUM(NSInteger, EOARateUsState)
 @property (nonatomic) OAApplicationMode* applicationMode;
 @property (nonatomic) OACommonString* availableApplicationModes;
 @property (nonatomic) OACommonAppMode* defaultApplicationMode;
+@property (nonatomic) OACommonAppMode* carPlayMode;
+@property (nonatomic) OACommonBoolean *isCarPlayModeDefault;
```
Comment (PaulStets) [inline Sources/Helpers/OAAppSettings.h:683]: I'm sure this can be done with only one setting.

Comment (nnngrach) [reply Sources/Helpers/OAAppSettings.h:683]: Don't do this

**Pull request merged**